python-sdk/src/astro/databases/snowflake.py

```diff
@@ -393,12 +392,12 @@ def create_stage(
         stage.set_url_from_file(file)
 
         fileformat = ASTRO_SDK_TO_SNOWFLAKE_FILE_FORMAT_MAP[file.type.name]
-        copy_options = [COPY_OPTIONS[file.type.name]]
+        copy_options = [COPY_OPTIONS.get(file.type.name)] if COPY_OPTIONS.get(file.type.name) else []
         copy_options.extend([f"{k}={v}" for k, v in self.load_options.copy_options.items()])
```

Comment:
I think we should log the final copy_options passed. Also, it would be nice if we can list the default options per-file basis in docs and/or log.

Reply:
added a debug log for whole  `COPY INTO` SQL statement

Reply:
https://github.com/astronomer/astro-sdk/pull/1689/commits/c4a6aec82c6a4e85d0bd1605c0fc77c0a0634447